mlflow/deployments/cli.py

```diff
@@ -0,0 +1,103 @@
+import click
+from mlflow.utils import cli_args
+from mlflow.deployments import interface
+from mlflow.deployments.utils import parse_custom_arguments
+
+
+deployment_target = click.option("--target", "-t", required=True, help="Deployment target")
+deployment_id = click.option("--id", "_deployment_id", required=True,
+                             help="Deployment ID for the deployment that needs to be deleted")
```

Comment:
```suggestion
                             help="ID of the deployment to be deleted")
```

We should probably define this option for each command vs globally, this help string makes sense for the `delete` command but not for `update`, where "ID of the deployment to be updated" would make more sense